addons/google_calendar/models/calendar.py

```diff
@@ -125,6 +125,9 @@ def _odoo_values(self, google_event, default_reminders=()):
             'videocall_location': google_event.get_meeting_url(),
             'show_as': 'free' if google_event.is_available() else 'busy'
         }
+        # Remove 'videocall_location' when not sent by Google, otherwise the local videocall will be discarded.
+        if not values.get('videocall_location'):
+            values.pop('videocall_location')
```

Comment:
Thanks for the change. I see an issue if Google decide to not provide a videocall_location key in the future.
values.get('videocall_location') will be None and values.pop('videocall_location') will trigger a KeyError exception. 
I am thinking about something like that:
```py
if values.get('videocall_location') is False:
    values.pop('videocall_location')
```
The traceback would prevent to sync events and it could block the application because we are syncing when we open it. As you are making changes in a stable version, would you mind add a small test in `addons/google_calendar/tests/test_sync_google2odoo.py` ?

Reply:
You can do `values.pop("videocall_location", False` for not getting error no matter if `values` contain the key or not.

Reply:
Tks @arj-odoo and @pedrobaeza I have fixed it and added a test for that.